packages/react-divider/src/components/Divider/useDividerStyles.ts

```diff
@@ -95,7 +91,20 @@ const useBaseStyles = makeStyles({
       ...shorthands.borderColor(tokens.colorBrandStroke1),
     },
   },
+  default: {
+    color: tokens.colorNeutralForeground2,
```

Comment:
Nit: You could break out the style changes to another PR to keep this one focused on just the commons change.

Reply:
I unfortunately cannot do that because this change also makes some of the commons props required in state (which they should have been all along tbh) and to keep the `styles[state.appearance]` work I need to make these style changes.